README.md

```diff
@@ -46,14 +46,18 @@ Help all ReScript Node.js apps and libraries to be built faster by reducing the
 ### Stream a file into stdout:
 
 ```rescript
+open NodeJs
+
 Fs.createReadStream("/path")
-  ->Stream.pipe(Process.(stdout(process)))
+  ->Stream.pipe(Process.stdout(Process.process))
```

Comment:
Local open is an error now, instead of just transformed into a block? damn. That's going to hurt my internal project which is still on v9.